go.mod

```diff
@@ -18,6 +18,8 @@ retract (
 // Internal deps fix version
 replace (
 	github.com/cihub/seelog => github.com/cihub/seelog v0.0.0-20151216151435-d2c6e5aa9fbf // v2.6
+	// add a "containerd_no_plugins" tag to remove the "plugin" import, which makes binaries bigger
+	github.com/containerd/containerd => github.com/pgimalac/containerd v1.7.24-0.20241227111440-c29d9741a30e
```

Comment:
What is the long term plan? 
* do an upstream contribution to add the solution? 
* keep a fork?

Reply:
If possible having this upstream would be best, I'll open an issue and PR to discuss it
But in the meantime I think the benefits far outweigh the drawbacks of having a fork

NB: as mentioned on the PR description, we have another possible solution, but I'm worried it could subtly change the behavior of other dependencies we have:
> Using the existing static_build tag would also work, but it is [used by many other packages](https://github.com/search?q=%22%21static_build%22+language%3AGo&type=code&l=Go&rgh-link-date=2024-12-27T11%3A31%3A47Z) so I was worried of unexpected behavior changes if we used it.